src/sync.cpp

```diff
@@ -4,9 +4,10 @@
 
 #include <sync.h>
 
+#include <map>
 #include <memory>
 #include <set>
-#include <util.h>
+#include <logging.h>
```

Comment:
nit: move the stdlib includes below (as is I believe clang-format will intermix them in the sort)